Sources/TuistKit/Generator/ProjectFileElements.swift

```diff
@@ -54,7 +54,14 @@ class ProjectFileElements {
                             sourceRootPath: sourceRootPath)
 
         /// Dependencies
-        generate(dependencies: graph.dependencies(path: project.path),
+        
+        func alwaysTrue(graphNode: GraphNode) -> Bool {
+            return true
+        }
+        
+        let dependencies = graph.findAll(path: project.path, test: alwaysTrue)
```

Comment:
Nitpick: What about inlining the closure:

```swift
let dependencies = graph.findAll(path: project.path, test: { _ in true })
```

Also we can use `@autoclosure` in the `test` argument and write it evens shorter:

```swift
let dependencies = graph.findAll(path: project.path, test: true)
```

_(Not sure if the code above would work though)_

Reply:
I hadn't thought about that, I like the implementation - let's go for it!

Reply:
I opted for just defining another function which defaults the test to always true

```
func findAll<T: GraphNode>(path: AbsolutePath) -> Set<T>
```